views/openacademy_res_partner_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="res_partner_view_form" model="ir.ui.view">
+        <field name="name">openacademy.res.partner.view.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <field name="active_lang_count" position="after">
```

Comment:
Could you use an xpath, please?

Note:  
Current approach is totally correct, I just wantto make sure you know the xpath syntax, as it's usually the most used one.